extensions/amp-selector/1.0/selector.type.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @externs */
+
+/** @const */
+var SelectorDef = {};
+
+/**
+ * @typedef {{
+ *   as: (string|PreactDef.FunctionalComponent),
+ *   disabled: (boolean|undefined),
+ *   value: (*|undefined),
```

Comment:
value is `Array`, so we should also be able to get rid of [this inline annotation](https://github.com/ampproject/amphtml/blob/f5096f882a7b9107773fbdd9a2c8d105cd47f867/extensions/amp-selector/1.0/selector.js#L39)

Reply:
I updated the type. But it should only be an array for a `multiple` mode, no? Is this a bug in our code?

Reply:
Not necessarily, value is an array in `0.1` for singular selectors too. We can modify to store the value as a non-array in our internal state but we would still have to expose it as an array of one item in the [AMP `select` event](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events/?referrer=ampproject.org#amp-selector).

Reply:
I see. I added it to tracker to decide on later. Let's keep as is for now.